examples/energy-management-app/esp32/main/main.cpp

```diff
@@ -117,10 +125,18 @@ namespace Clusters {
 namespace DeviceEnergyManagement {
 
 // Keep track of the parsed featureMap option
+#if 1 // 0x7a
 static chip::BitMask<Feature> sFeatureMap(Feature::kPowerAdjustment, Feature::kPowerForecastReporting,
+                                          Feature::kStartTimeAdjustment, Feature::kPausable,
+                                          Feature::kForecastAdjustment, Feature::kConstraintBasedAdjustment);
+#else // 0x7c
+static chip::BitMask<Feature> sFeatureMap(Feature::kPowerAdjustment,
                                           Feature::kStateForecastReporting, Feature::kStartTimeAdjustment, Feature::kPausable,
                                           Feature::kForecastAdjustment, Feature::kConstraintBasedAdjustment);
 
+static const char * spApp = nullptr;
```

Comment:
Not clear why this is in this branch of the #if / #else ?

Reply:
It is obsolete as now control the app type using conditional compilation.